packages/vue/src/Form/MultipleUploadFile/OcMultipleUploadFile.vue

```diff
@@ -21,6 +21,7 @@ const props = defineProps({
     type: String,
     default: "",
   },
+  validateAcceptFileType: Boolean,
```

Comment:
what does this `validateAcceptFileType` prop relate to ? would this be better named as `validateFileType`? I find it difficult to understand `validateAcceptFileType` ?